editor/src/components/inspector/resize-to-fit-control.spec.browser2.tsx

```diff
@@ -261,3 +346,59 @@ export var storyboard = (
   </Storyboard>
 )
 `
+
+const projectWithGroup = `import * as React from 'react'
```

Comment:
nitpick but I think the test is more readable if both the before and after code is in the test body. but it's better if it's not the entire project code, just the relevant middle snippet. 

but at the same time the rest of the test file is structured like this, so I get the argument that this new test just follows the (admittedly suboptimally readable) pattern